pkg/cluster/config.go

```diff
@@ -4,12 +4,14 @@ import (
 	"context"
 	"fmt"
 	"net"
+	"strconv"
 	"strings"
 
 	oconfig "github.com/openshift/api/config/v1"
 	configclient "github.com/openshift/client-go/config/clientset/versioned"
 	operatorv1 "github.com/openshift/client-go/operator/clientset/versioned/typed/operator/v1"
 	"github.com/pkg/errors"
```

Comment:
We should really consider using the [stdlib errors package](https://golang.org/pkg/errors/) instead of this one

Reply:
not a problem for you to solve of course :smile:

Reply:
I could do it in a separate commit

Reply:
The wrapping part is just `fmt.Errorf("%w: ...", err, ...))`

But the reasoning would be code purity and reducing the amount of imports. No functional improvements as far as I'm aware. In the future language improvements might make that case though.

Reply:
This needs to be done as part of separate PR since it include changes in almost every file in `/pkg` and `/test` modules